packages/concerto-core/test/data/decoratorcommands/extract-test.cto

```diff
@@ -18,20 +18,26 @@ asset assetName identified by assetKey {
     o String assetKey
 }
 
+@Term("Map Name")
 map mapName {
+    @Term("some key")
+    @Term_desc("some key description")
     @deco(1)
     o String
     o String
 }
 
 @Term("Person Class")
+@Term_desc("Person Class Description")
 @Editable
 concept Person {
     @Term("HI")
     @Custom
     @Form("inputType", "text")
     @New
     o String firstName
+    @term("custom")
```

Comment:
I have intentionally added "term" for testing purpose. It should be considered as non-vocab.

Reply:
Does the test capture this? The extracted vocab does not include the `term`?
```
expect(vocab).should.not.have('cusotm') // or something like this
```

Reply:
I think what I meant is that shouldn't we have an explicit check to see if extracted vocabulary does not have vocabulary term from `@term` and dcs extracted has that included as a decorator?

Reply:
expect(vocab).should.not.have('custom')
expect(dcs.commands[0].decorator.name).should.equals('custom')
// this kind of assertion in either of the test is missing. This can make someone who is new more clear about `term` vs `Term`.

Reply:
Got it. Updated.